### DISCUSSION TITLE
Cant login admin on umbrel

### DISCUSSION THREAD

**Discussion by claudiodb007** [balzack/databag#85] [Q&A]
Hi, Installed databag using umbrel but cant login, always wrong password. used the default password showing on umbrel dashboard but dosent work

how can i change/reset administrator password ?

already tried uninstall and install again

Thank you

**Comment by balzack** [comment]
Hello, the default password provided by umbrel is for the admin dashboard. (upper right cog icon)

From the admin dashboard you can create new accounts for which you will assign a username and password.

Please let me know if I misunderstood your issue.
